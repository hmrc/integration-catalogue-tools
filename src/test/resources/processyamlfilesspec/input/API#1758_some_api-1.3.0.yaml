openapi: 3.0.3
info:
  title: API#1758
  description: >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
  version: 1.2.1
  contact:
    name: Joe Bloggs
servers:
  - url: https://{hostname}:{port}
    description: Different hostnames and ports are used in different environments
    variables:
      hostname:
        default: localhost
        description: The hostname
      port:
        default: '443'
        description: The port
