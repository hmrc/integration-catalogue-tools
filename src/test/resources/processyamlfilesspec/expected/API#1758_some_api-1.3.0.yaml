openapi: 3.0.3
info:
  title: API#1758
  description: >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
  version: 1.2.1
  contact:
    name: Joe Bloggs
  x-integration-catalogue:
    reviewed-date: 2022-04-22T20:27:05Z
    platform: CORE_IF
    publisher-reference: 1758
servers:
  - url: https://{hostname}:{port}
    description: Different hostnames and ports are used in different environments
    variables:
      hostname:
        default: localhost
        description: The hostname
      port:
        default: '443'
        description: The port
